### PR TITLE
Upgrade to celery 4.1

### PR DESCRIPTION
**Pull request by Ladsgroup**
\o/

Bug: T178441

### CODE REVIEW

File: ores/scoring_systems/celery_queue.py [hunk 1]
```diff
@@ -24,13 +24,13 @@
 
 
 @before_task_publish.connect
-def update_sent_state(sender=None, body=None, **kwargs):
```
[the remark this answers is not in the document]
Comment (Ladsgroup) [reply ores/scoring_systems/celery_queue.py]: Completely removed now :dagger:

**Comment by Ladsgroup** [comment]
I tested this in ores.wmflabs.org and it works fine in so many levels. Including recovery after server overload.

**Comment by halfak** [comment]
How did you test for an overload?  How do you know that the service became overloaded?  How do you know that it recovered as expected?

**Comment by Ladsgroup** [comment]
> How did you test for an overload?
I ran a precache + 100 parallel connections to get random scores (poolcounter doesn't work here anymore)
> How do you know that the service became overloaded?
Gave me 503 for all request.
> How do you know that it recovered as expected?
I stopped the precache and ran it with less parallel connections and it started to respond with data.

**Pull request merged**